app/policies/topic_policy.rb

```diff
@@ -0,0 +1,2 @@
+class TopicPolicy < ApplicationPolicy
```

Comment:
Why are you creating a policy? It seems topics would be accessible without authentication, wdyt?

Reply:
I created a policy because when I try to call the API to obtain the topics, I receive this error: Pundit::PolicyScopingNotPerformedError.